ProcessQueue/__init__.py

```diff
@@ -215,19 +233,20 @@ def main(msg: func.QueueMessage):
         # Create a dictionary with field names and field ids 
         # as key value pairs
         # e.g ("Field Name":<Field ID>)
-        namexids=dict(zip(field_names,field_ids))
-
+        namexids=dict(zip(field_ids,field_names))
+        
         # For sheets past the first two in the Scan Report
         # i.e. all 'data' sheets that are not Field Overview and Table Overview
         data=[]

```

Comment:
Please remove as this is unused.

Reply:
This is used on line 272 to match the values to the fields

Reply:
Sorry, I was referring to `data` which is reset within the loop below.

Reply:
ah yeah you mean I'm emptying it again as I enter the for loop on line 243